mappings/net/minecraft/client/render/entity/model/SinglePartEntityModel.mapping

```diff
@@ -23,4 +25,4 @@ CLASS net/minecraft/unmapped/C_kzsusdgk net/minecraft/client/render/entity/model
 		ARG 3 animationProgress
 	METHOD m_tcdltrvc (Lnet/minecraft/unmapped/C_mnzbondw;Lnet/minecraft/unmapped/C_kxntavoz;)V
 		ARG 2 _state
-	METHOD m_vcfjyysm getPart ()Lnet/minecraft/unmapped/C_rglqxnbw;
+	METHOD m_vcfjyysm getBody ()Lnet/minecraft/unmapped/C_rglqxnbw;
```

Comment:
why?

Reply:
I re-reviewed some changes I made when mapping this, and I realized that I mapped this incorrectly as I had misnamed some attributes related to the Breeze entity. I missed that it was an override to a method from an inherited class. Fixed that, and I appreciate the catch!